kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -134,16 +136,16 @@ def check(self, instance):
 
         # Report the broker highwater offset
         for (topic, partition), highwater_offset in highwater_offsets.iteritems():
-            broker_tags = ['topic:%s' % topic, 'partition:%s' % partition]
+            broker_tags = ['topic:%s' % topic, 'partition:%s' % partition] + custom_tags
             self.gauge('kafka.broker_offset', highwater_offset, tags=broker_tags)
 
         # Report the consumer group offsets and consumer lag
         if zk_consumer_offsets:
             self._report_consumer_metrics(highwater_offsets, zk_consumer_offsets,
-                                          topic_partitions_without_a_leader, ['source:zk'])
+                                          topic_partitions_without_a_leader, tags=custom_tags+['source:zk'])
         if kafka_consumer_offsets:
             self._report_consumer_metrics(highwater_offsets, kafka_consumer_offsets,
-                                          topic_partitions_without_a_leader, ['source:kafka'])
+                                          topic_partitions_without_a_leader, tags=custom_tags+['source:kafka'])
```

Comment:
Could we add white space around the operators?

Reply:
done!